index.html

```diff
@@ -32,23 +32,157 @@ <h3>
       </h3>
       <span class="emphasize no-decoration"><a href="#">LET&rsquo;S CONNECT</a></span>
       <ul class="social-links">
-        <li class="twitter">
+        <li>
           <a href="#" title="Twitter"><i class="fa-brands fa-twitter no-decoration"></i></a>
         </li>
-        <li class="linkedin">
+        <li>
           <a href="#" title="Linked-in"><i class="fa-brands fa-linkedin-in no-decoration"></i></a>
         </li>
-        <li class="medium">
+        <li>
           <a href="#" title="Medium"><i class="fa-brands fa-medium no-decoration"></i></a>
         </li>
-        <li class="github">
+        <li>
           <a href="#" title="GitHub"><i class="fa-brands fa-github no-decoration"></i></a>
         </li>
-        <li class="angellist">
+        <li>
           <a href="#" title="AngelList"><i class="fa-brands fa-angellist no-decoration"></i></a>
         </li>
       </ul>
     </section>
+
+    <div class="works">
+      <article>
+        <a href="#"><img src="assets/work_1.png" alt="Preview of this project"></a>
+        <h2><a href="#">Tonic</a></h2>
+        <div class="info">
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
```

Comment:
- [ ] Kindly use the ul tag to group them and use li for each content not the `a` tag.